Makefile

```diff
@@ -106,7 +106,10 @@ define lint-common
 	echo "Running linting for $(2)..."
 	@(cd $(1) && poetry run ruff format $(2))
 	@(cd $(1) && poetry run ruff check --fix $(2))
+	echo "Running mypy for $(2)..."
 	@(cd $(1) && poetry run mypy $(2))
```

Comment:
Since mypy has the benefit of the official pydantic plugin, and sometimes flags different errors than pyright, we keep both of them in the linting pipeline.